docs/03-tutorials/00-serverless/svls-05-debug-function.md

```diff
@@ -40,6 +40,42 @@ Follows these steps:
     kyma run function --debug
     ```
 
+</details>
+<details>
+  <summary label="vsc">
+  Visual Studio Code (Python)
+  </summary>
+
+1. In VSC, navigate to the location of the file with the Function definition.
+2. Create the `.vscode` directory.
+3. In the `.vscode` directory, create the `launch.json` file with this content:
+   ```json
+   {
+      "version": "0.2.0",
+      "configurations": [
+          {
+              "name": "Python: Kyma function",
+              "type": "python",
+              "request": "attach",
+              "pathMappings": [
+                  {
+                      "localRoot": "${workspaceFolder}",
+                      "remoteRoot": "/kubeless"
+                  }
+              ],
+              "connect": {
+                  "host": "localhost",
+                  "port": 5678
+              }
+          }
+      ]
+  }
+  ```
+4. Run the Function with the `--debug` flag.
+    ```bash
+    kyma run function --debug
+    ```
+
```

Comment:
This part does not render correctly and affects the rendering of the next dropdown menu (GoLand), too. You can see it by choosing **View file**. Improving indentation should solve the problem.

Reply:
Thanks for reviewing it @IwonaLanger. I've fixed the formatting. Apologies for not noticing this before.

Reply:
> Thanks for reviewing it @IwonaLanger. I've fixed the formatting. Apologies for not noticing this before.

No worries :)